pyproject.toml

```diff
@@ -17,4 +17,5 @@ dependencies = [
     "email-validator",
     "icalendar",
     "xlsxwriter",
+    "pytest>=8.3.5",
```

Comment:
This should go into an `optional-dependencies` section, because the program doesn't need pytest at runtime.